src/Dfe.PlanTech.Domain/Persistence/Interfaces/IWebhookToDbCommand.cs

```diff
@@ -0,0 +1,20 @@
+using Dfe.PlanTech.Domain.ServiceBus.Models;
+
+namespace Dfe.PlanTech.Domain.Persistence.Interfaces
+{
+    /// <summary>
+    /// Processes Contentful webhook payloads, and saves them to the DB
+    /// </summary>
+    public interface IWebhookToDbCommand
+    {
+        /// <summary>
+        /// 
```

Comment:
👀 can probs drop the empty summary, or put something in it

Reply:
What was I even doing here? I was clearly going to add it and then just... didn't? Finished! Thank you